Spring-Project/src/main/java/com/epam/project/spring/model/User.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.project.spring.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class User {
+
+    private Integer id;
```

Comment:
Hi Zenyk. Please process '**id**' field on the **Repository** level in the same manner as for the **Product** entity. Thanks

Reply:
Thanks. Resolved